arroyo-console/src/routes/home/Home.tsx

```diff
@@ -48,27 +48,24 @@ export const Stat = (props: Props) => {
   );
 };
 
-export function Home({ client }: { client: ApiClient }) {
-  const [state, setState] = useState<HomeState>({ jobs: null });
-
-  useEffect(() => {
-    const fetchData = async () => {
-      const clientObj = await client();
-      const jobs = clientObj.getJobs(new GetJobsReq({}));
-
-      setState({
-        jobs: (await jobs).jobs,
-      });
-    };
+const fetchJobs = async (client: ApiClient) => {
+  const clientObj = await client();
+  return (await clientObj.getJobs(new GetJobsReq({}))).jobs;
+};
 
-    fetchData();
-  }, []);
+export function Home({ client }: { client: ApiClient }) {
+  const { data: jobs } = useSWR('jobs', () => fetchJobs(client));
```

Comment:
I suggest we keep all `useSWR` calls in `data_fetching.ts`. Doing so keeps all the api client interaction in one place and makes it easier to trigger refreshes of dependent api calls.

Reply:
Fixed with https://github.com/ArroyoSystems/arroyo/pull/206/commits/1f76c8bd2bfb017c97c38ae2d182964a169776e8